host/bcm283x/gpio.go

```diff
@@ -325,13 +328,18 @@ func (p *Pin) FastOut(l gpio.Level) {
 //
 // PWM pins
 //
-// PWM0 is exposed on pins 12, 18 and 40.
+// PWM0 is exposed on pins 12, 18 and 40. However, PWM0 is used for generating
+// clock for DMA and unavailable for PWM.
+// TODO(simokawa): Use other DREQ source like PCM.
```

Comment:
We can't use PCM as a source, we'll use it for binary streaming, so remove the TODO here.

Reply:
It depends on application. Anyway, I have no plan to work on it in the short term, removed it.